app/src/main/java/pl/example/android/garbageapp/data/database/SectorTermsDatabaseDataSource.java

```diff
@@ -62,19 +63,20 @@ public void scheduleCountingSectorTermsForNotification() {
     }
 
     public void countSectorTermsForNotification() {
-        Date today = SectorTermsDateUtils.getNormalizedUtcDateForToday();
-        int count = mSectorTermDao.countSectorTermsForToday(today);
-        if (count > 0) {
-            NotificationUtils.showNotification(mContext, "Dziś wywóz śmieci");
+        Date tomorrow = SectorTermsDateUtils.getNormalizedUtcDateForTomorrow();
+        int count = mSectorTermDao.countSectorTermsForTomorrow(tomorrow);
```

Comment:
One important is missing here. We should take SectorColor from SharedPreferences. In case SectorColor is not set do not call DAO method at all.